projects/archiver/src/utils/s3.ts

```diff
@@ -28,10 +28,17 @@ const createCMSFrontsS3Client = () => {
     })
 }
 
-export const s3 = new S3({
+const s3DefaultCredentials = new S3({
     region: 'eu-west-1',
 })
 
+const s3FrontendCredentials = new S3({
+    region: 'eu-west-1',
+    credentials: new SharedIniFileCredentials({ profile: 'frontend' }),
+})
+
+export const s3 = process.env.arn ? s3DefaultCredentials : s3FrontendCredentials
```

Comment:
Looks like this fail to run. 
I have tested it and it fails to run because `process.env.arn` is undefined when it runs on aws.

Any idea why?

Reply:
Ah it failed because `arn` doesn't seem to exist on every bit of the archiver. I've made this better (and easier to understand) now and just switched based off the stage variable